performance/include/traccc/utils/helpers.hpp

```diff
@@ -0,0 +1,81 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include <string>
+
+#include "TEfficiency.h"
+#include "TFitResult.h"
+#include "TFitResultPtr.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TProfile.h"
+#include "TROOT.h"
+
+namespace traccc {
+
+namespace plot_helpers {
+/// @brief Nested binning struct for booking plots
+struct binning {
+    binning();
+
+    binning(std::string b_title, int bins, float b_min, float b_max);
```

Comment:
Hmm let me do this in the separate PR. I don't have any objection with that but this will require change in `eff_plot_tool.hpp` and `duplicate_plot_tool.hpp` as well.